ipware/defaults.py

```diff
@@ -57,7 +57,6 @@
         '233.', '234.', '235.', '236.', '237.', '238.', '239.',  # multicast
         '240.', '241.', '242.', '243.', '244.', '245.', '246.', '247.', '248.',
         '249.', '250.', '251.', '252.', '253.', '254.', '255.',  # reserved
-    ) + (
```

Comment:
Why are we removing this line?

Reply:
<img width="1236" alt="Screenshot 2023-07-27 at 4 11 30 PM" src="https://github.com/irtazaakram/django-ipware/assets/51848298/86e16138-e61d-4164-91bd-239a53571a98">